src/components/source-info.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+/**
+ * @typedef SourceInfo
+ * @property {Object} from
+ * @property {Object} to
+ * @property {String} pageUrl
+ */
+
+/**
+ * Renders link to see Wayback Machine calendar view of the page
+ * If one or both of the page diffs were sourced from Wayback, this component renders links to the Wayback Machine source url(s)
+ *
+ * @class SourceInfo
+ * @extends {React.Component}
+ * @param {SourceInfoProps} props
+ */
+
+const sourceTypeName = {
+  internet_archive: 'Wayback Machine'
+};
+
+export default class SourceInfo extends React.Component {
+  render () {
+    const waybackCalendarUrl = `https://web.archive.org/web/*/${this.props.pageUrl}`;
+    
+    return (
+      <div>
+        <i className="fa fa-calendar" aria-hidden="true" />
```

Comment:
> Also not sure if it would be better for accessibility to do one or both of:
> 
> * Use `<aside>` instead of `<div>`.
> * Use a list (e.g. `<ol>`) for the links instead of just text.

Great point! I think both are helpful in terms of semantics + navigation in this case.